python/ray/data/tests/test_dataset_image.py

```diff
@@ -38,6 +38,23 @@ def test_multiple_paths(self, ray_start_regular_shared):
         )
         assert ds.count() == 2
 
+    @pytest.mark.parametrize("ignore_missing_paths", [True, False])
+    def test_ignore_missing_paths(self, ray_start_regular_shared, ignore_missing_paths):
+        paths = [
+            "example://image-datasets/simple/image1.jpg",
+            "example://missing.jpg",
```

Comment:
let's also add a test for missing directory, e.g. `example://image-datasets/missing/`

Reply:
added